assets/js/base/components/price-slider/style.scss

```diff
@@ -263,27 +263,31 @@
 		}
 	}
 
-	&.is-loading,
-	&.is-disabled {
-		.wc-block-components-price-slider__range-input-wrapper {
-			@include placeholder();
-			box-shadow: none;
+	.wc-block-components-price-slider {
+		&.is-loading,
+		&.is-disabled {
```

Comment:
While not related to this PR, I noticed CSS selectors were missing the `.wc-block-components-price-slider` parent. It should be fixed now.